components/textdiff/textdiff.js

```diff
@@ -198,7 +206,7 @@ class TextDiffViewModel {
     }
     return `<div class="d2h-code-line-prefix"><span data-bind="visible: editState() !== 'patched'">${symbol}</span><input ${
       isActive ? 'checked' : ''
```

Comment:
This can be removed.

Also the [`isActive` check](https://github.com/FredrikNoren/ungit/pull/1429/files#diff-21bc09960eb0842e91cc5dfecb8efa75R204) could go [here](https://github.com/FredrikNoren/ungit/pull/1429/files#diff-21bc09960eb0842e91cc5dfecb8efa75R178) so `getPatchCheckbox` has no sideeffects.

Reply:
Okay, this is done.  See 74b679f and 051d956.